internal/util/dirent.go

```diff
@@ -20,6 +20,10 @@ func RemoveDirent(path string) error {
 	return err
 }
 
+func CopyDirentDefault(source string, dest string) error {
```

Comment:
I would flip this and instead use `CopyDirent(source, dest)` and `CopyDirentFull(source, dest, onlyContents)`.
This would be nice, since most people won't have to think about onlyContents.